src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days === 7) {
+    return 230;
+  }
+
+  if (days === 6) {
+    return 220;
+  }
+
+  if (days === 3) {
+    return 100;
+  }
+
+  return 80;
```

Comment:
The current implementation only checks for specific numbers of days (7, 6, and 3) and returns a fixed cost for these. For any other number of days, it defaults to returning 80. This might not be the intended behavior if you want to calculate rental costs for a wider range of days. Consider implementing a more flexible pricing strategy that accounts for different day ranges or a per-day rate.